src/vcf-pdf-viewer.js

```diff
@@ -466,6 +456,121 @@ class PdfViewerElement extends
         };
     }
 
+    /**
+     * Adds toggle button to the toolbar slot named "sidebar-toggle-button-slot".
+     */
+    _createSideBarToggleButton() {
```

Comment:
Extract a reusable button creation method: The code for creating the buttons is highly repetitive.

Reply:
Done. See __createToolbarButton() method.